crates/apub/src/activities/send/community.rs

```diff
@@ -230,18 +258,57 @@ impl CommunityType for Community {
     removed_mod: Person,
     context: &LemmyContext,
   ) -> Result<(), LemmyError> {
-    let mut remove = Remove::new(
-      actor.actor_id.clone().into_inner(),
-      removed_mod.actor_id.into_inner(),
-    );
+    let mut remove = Remove::new(actor.actor_id(), removed_mod.actor_id());
     remove
       .set_many_contexts(lemmy_context()?)
       .set_id(generate_activity_id(RemoveType::Remove)?)
       .set_to(public())
       .set_many_ccs(vec![self.actor_id()])
       .set_target(generate_moderators_url(&self.actor_id)?.into_inner());
 
-    send_to_community(remove, &actor, self, context).await?;
+    send_to_community(remove, &actor, self, Some(removed_mod.actor_id()), context).await?;
+    Ok(())
+  }
+
+  async fn send_block_user(
+    &self,
+    actor: &Person,
+    blocked_user: Person,
+    context: &LemmyContext,
+  ) -> Result<(), LemmyError> {
+    let mut block = Block::new(actor.actor_id(), blocked_user.actor_id());
```

Comment:
Makes sense, never realized a ban is basically a community blocking a user. This might be usable for the eventual user <-> user blocks too.